osgp/platform/osgp-domain-core/src/main/java/org/opensmartgridplatform/domain/core/valueobjects/smartmetering/GetPowerQualityProfileRequest.java

```diff
@@ -95,12 +95,12 @@ public boolean equals(final Object obj) {
         if (this == obj) {
             return true;
         }
-        if (!(obj instanceof ProfileGenericDataRequest)) {
+        if (!(obj instanceof GetPowerQualityProfileRequest)) {
             return false;
         }
-        final ProfileGenericDataRequest other = (ProfileGenericDataRequest) obj;
+        final GetPowerQualityProfileRequest other = (GetPowerQualityProfileRequest) obj;
         return Objects.equals(this.deviceIdentification, other.deviceIdentification)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1067)